tests/integration/ovs_test.py

```diff
@@ -144,6 +144,22 @@ def test_vlan_as_ovs_bridge_slave(vlan_on_eth1):
         assertlib.assert_state_match(state)
 
 
+@pytest.mark.xfail(
+    raises=(NmstateLibnmError, AssertionError),
+    reason="https://bugzilla.redhat.com/1724901",
+)
```

Comment:
This is not needed right? The test should pass.. Am I missing something? Thanks!

Reply:
https://bugzilla.redhat.com/1724901 is about OVS instability in general, it waits for NM to fix their other problems

Reply:
Oh, sorry. Looks good then